src/lib/requests.ts

```diff
@@ -33,13 +33,13 @@
 // Testing will be covered in PR #11
 
 import { config } from './config'
-// @ts-ignore
-import { Logger, ThirdpartyRequests, BaseRequestConfigType } from '@mojaloop/sdk-standard-components'
+import { ThirdpartyRequests, BaseRequestConfigType } from '@mojaloop/sdk-standard-components'
+import Logger from '@mojaloop/central-services-logger'
 
 // Config file to instantiate ThirdPartyRequest object
 const configRequest: BaseRequestConfigType = {
   dfspId: config.get('PARTICIPANT_ID') as string,
```

Comment:
if you want to have string at execution time use `.toString()` typecasting will work only at compilation time

Reply:
https://stackoverflow.com/questions/47664576/as-string-versus-tostring

Reply:
please check and fix if used anywhere in source code files you have touched

Reply:
@eoln this already exists in the code base. I have not changed anything related to the string typecasting of config fields. It came from @akapur99 's PR. @akapur99 maybe you could have a look at this?